template/src/__tests__/App-test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import App from '../App';
+import renderer from 'react/lib/ReactTestRenderer';
+
+describe('App', () => {
+  it('renders a welcome view', () => {
+    const instance = renderer.create(<App />);
+    const tree = instance.toJSON();
```

Comment:
Is this API going to change? I remember there were talks about not wanting people to inspect the JSON.